internal/engine/entities/entity_event.go

```diff
@@ -287,6 +298,13 @@ func (eiw *EntityInfoWrapper) withProviderFromMessage(msg *message.Message) erro
 	return nil
 }
 
+//nolint:unparam
```

Comment:
Why is this needed? It looks like the argument and the method receiver are both used.

Reply:
Yeah, it was because we always returned nil as an error. I had as I thought to enforce having it but forgot to remove it. Thanks for noticing this!